src/utilcode/prettyprintsig.cpp

```diff
@@ -335,15 +335,10 @@ static PCCOR_SIGNATURE PrettyPrintType(
                 {       
                     if (sizes[i] != 0 && lowerBounds[i] != 0)   
                     {   
-                        if (lowerBounds[i] == 0)        
-                            appendStrNumW(out, sizes[i]);
-                        else    
-                        {       
-                            appendStrNumW(out, lowerBounds[i]);
-                            appendStrW(out, W("..."));
-                            if (sizes[i] != 0)  
-                                appendStrNumW(out, lowerBounds[i] + sizes[i] + 1);
-                        }       
+                        appendStrNumW(out, lowerBounds[i]);
+                        appendStrW(out, W("..."));
+                        if (sizes[i] != 0)  
```

Comment:
The enclosing if test ensures `sizes[i] != 0` here and below.

Reply:
removed that check also